### ISSUE TITLE
Bug: Branch angle difference violation compares radians against degrees

### ISSUE THREAD

**Issue by naomi-simumba** [gridfm/gridfm-graphkit#98]
The branch angle difference violation computation in `OptimalPowerFlowTask.test_step` compared `angle_diff` (in radians) directly against `ANG_MIN` and `ANG_MAX` (in degrees), producing incorrect violation magnitudes.

*Cause*
In the [normalizers.py](https://github.com/gridfm/gridfm-graphkit/blob/f71bcccbeeff3b89f5cd006ac5b97876dbcc4761/gridfm_graphkit/datasets/normalizers.py#L186) script, `HeteroDataMVANormalizer.transform()` converts both voltage angles and branch angle limits from degrees to radians when loading the dataset (same pattern in `HeteroDataPerSampleMVANormalizer.transform()`):

```        
        data.x_dict["bus"][:, VA_H] *= torch.pi / 180.0
```

```
        data.edge_attr_dict[("bus", "connects", "bus")][:, ANG_MIN] *= torch.pi / 180.0
        data.edge_attr_dict[("bus", "connects", "bus")][:, ANG_MAX] *= torch.pi / 180.0
```

The `inverse_transform()` reverses `ANG_MIN`/`ANG_MAX` back to degrees but intentionally leaves `VA_H` and model output angles in radians, because the physics layers (`ComputeBranchFlow`, `ComputeNodeResiduals`) expect radians:

```
       data.edge_attr_dict[...][..., ANG_MIN] *= 180.0 / torch.pi  
       data.edge_attr_dict[...][..., ANG_MAX] *= 180.0 / torch.pi  
```

Finally, the `OptimalPowerFlowTask.test_step` calls `inverse_transform()`and `inverse_output()` at the start so that when the violation check runs ([here](https://github.com/gridfm/gridfm-graphkit/blob/f71bcccbeeff3b89f5cd006ac5b97876dbcc4761/gridfm_graphkit/tasks/opf_task.py#L142)),  the relu is comparing radians against degrees.

- `angle_min` and `angle_max` (from `ANG_MIN` / `ANG_MAX` in `bus_edge_attr`) are in degrees 
- `angle_diff`(from `VA_OUT` model output) is in radians.

```
       angle_excess_low = F.relu(angle_min - angle_diff)
       angle_excess_high = F.relu(angle_diff - angle_max)
```

*Proposed Solution*
Convert `ANG_MIN` / `ANG_MAX` in `bus_edge_attr` back to radians prior to comparison

Addressed is PR: https://github.com/gridfm/gridfm-graphkit/pull/99

**Comment by naomi-simumba** [comment]
Resolved in PR #99 and merged
